config/dev_config.yml

```
@@ -40,6 +40,7 @@ github:
 
 IPScan:
   contactEmail: interpro-team@ebi.ac.uk
+  lowPriorityEmail: interpro-team-normal-priority@ebi.ac.ukß
```

Introducing German letters to celebrate Oktoberfest? :de: :beers: 

```suggestion
  lowPriorityEmail: interpro-team-normal-priority@ebi.ac.uk
```